assets/js/plugin-check-admin.js

```diff
@@ -1,24 +1,30 @@
 ( function ( pluginCheck ) {
 	const checkItButton = document.getElementById( 'plugin-check__submit' );
+	const resultsContainer = document.getElementById( 'plugin-check__results' );
 	const pluginsList = document.getElementById(
 		'plugin-check__plugins-dropdown'
 	);
 
 	// Return early if the elements cannot be found on the page.
-	if ( ! checkItButton || ! pluginsList ) {
+	if ( ! checkItButton || ! pluginsList || ! resultsContainer ) {
 		console.error( 'Missing form elements on page' );
 		return;
 	}
 
 	checkItButton.addEventListener( 'click', ( e ) => {
 		e.preventDefault();
 
+		// Empty the results container.
+		resultsContainer.innerText = '';
+
 		getChecksToRun()
 			.then( setUpEnvironment )
 			.then( runChecks )
 			.then( cleanUpEnvironment )
 			.then( ( data ) => {
 				console.log( data.message );
+
+				resultsContainer.innerHTML += 'Checks complete';
```

Comment:
This should be a translated string, also we should probably put it at least in some minimal markup, like a `<p>` tag.

Last but not least, having this line here seems out of context. Shouldn't it rather happen as the last bit of `renderResults()` as it is connected to that function?